src/menu/settings_menu.rs

```diff
@@ -772,6 +815,47 @@ impl SettingsMenu {
                 }
                 _ => (),
             },
+            CurrentMenu::LinksMenu => match self.links.tick(controller, state) {
+                MenuSelectionResult::Selected(LinksMenuEntry::DiscordLink, _) => {
+                    if let Err(e) = webbrowser::open(DISCORD_LINK) {
+                        log::warn!("Error opening web browser: {}", e);
+                    }
+                }
+                MenuSelectionResult::Selected(LinksMenuEntry::GithubLink, _) => {
+                    if let Err(e) = webbrowser::open(GITHUB_LINK) {
+                        log::warn!("Error opening web browser: {}", e);
+                    }
+                }
+                MenuSelectionResult::Selected(LinksMenuEntry::DocsLink, _) => {
+                    if let Err(e) = webbrowser::open(DOCS_LINK) {
+                        log::warn!("Error opening web browser: {}", e);
+                    }
+                }
+                MenuSelectionResult::Selected(LinksMenuEntry::TributeLink, _) => {
+                    if let Err(e) = webbrowser::open(TRIBUTE_LINK) {
+                        log::warn!("Error opening web browser: {}", e);
+                    }
+                }
+                MenuSelectionResult::Selected(LinksMenuEntry::GeneralLink, _) => {
+                    if let Err(e) = webbrowser::open(GENERAL_LINK) {
+                        log::warn!("Error opening web browser: {}", e);
+                    }
+                }
+                MenuSelectionResult::Selected(LinksMenuEntry::ModdingLink, _) => {
+                    if let Err(e) = webbrowser::open(MODDING_LINK) {
+                        log::warn!("Error opening web browser: {}", e);
+                    }
+                }
+                MenuSelectionResult::Selected(LinksMenuEntry::GetplusLink, _) => {
+                    if let Err(e) = webbrowser::open(GETPLUS_LINK) {
+                        log::warn!("Error opening web browser: {}", e);
+                    }
+                }
```

Comment:
...and finally these repetitive match arms can be simplified to:
```rs
                MenuSelectionResult::Selected(LinksMenuEntry::Link(url), _) => {
                    if let Err(e) = webbrowser::open(&url) {
                        log::warn!("Error opening web browser: {}", e);
                    }
                }
```

Reply:
all set, thanks for the help